pkg/observer/data_stats.go

```diff
@@ -0,0 +1,36 @@
+package observer
+
+import (
+	"github.com/cilium/tetragon/pkg/metrics/consts"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	// Define a counter metric for data event statistics
+	DataEventStats = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name:        consts.MetricNamePrefix + "data_event_stats",
```

Comment:
Can you add a `_total` suffix to the counter, for compatibility with the [OpenMetrics](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#suffixes) conventions?